plugin/src/main/java/org/wildfly/swarm/plugin/maven/RepositorySystemSessionWrapper.java

```diff
@@ -175,4 +190,24 @@ public RepositoryCache getCache() {
     private RepositorySystemSession delegate;
 
     private DependencyGraphTransformer transformer;
+
+    private boolean excludeSwarm;
+
+    public class SwarmExcludedTraverser implements DependencyTraverser
+    {
+
+       @Override
+       public boolean traverseDependency(Dependency dependency)
+       {
+          return !SWARM_GROUP_ID.equals(dependency.getArtifact().getGroupId());
```

Comment:
@bobmcwhirter At this stage during maven processing the file is not available. Any other suggestions?